docs/index.md

```diff
@@ -122,6 +130,21 @@ $ docker run \
 !!! note
     See documentation or add the `--help` flag for more.
 
+## Quickstart (kubernetes)
+
+Tracee is designed for use in Kubernetes deployments so the process for doing so is straightforward.
+
+1. Install tracee via helm.
+
+See the kubernetes [installation guide].
+
+2. FIXME:
+
+- How are users interacting with Tracee via Kubernetes?
+  - Are signature alerts printed to stderr and then collected via logs?
+  - Is Postee enabled by default and if so how is it configured?
+  - How are signatures configured?
```

Comment:
@josedonizetti @simar7 Perhaps you two can help me with this?

Reply:
sorry don't have a context but I'll share what I know about tracee + k8s:
- tracee is deployed via a helm chart https://github.com/aquasecurity/tracee/blob/main/deploy/helm/tracee/values.yaml
- this helm chart has an included dependency on postee, which is how today people can see their tracee rules events showing up in the form of an UI.
- there's nothing special in terms of configuring signatures, the default ones are loaded to be run.

Reply:
@grantseltzer @simar7 I think we should make postee not enabled by default. The first interaction of a user with tracee would be through stdout logs. Than using postee+tracee should be a tutorial, where we can dive more into postee usage. 

Yes, currently all signatures will be loaded by default, and there isn't a way to configure it.